components/com_fields/src/Controller/FieldController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * @package     Joomla.Site
+ * @subpackage  com_fields
+ *
+ * @copyright   (C) 2017 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Fields\Site\Controller;
+
+\defined('_JEXEC') or die;
+
+use Joomla\CMS\Response\JsonResponse;
+
+/**
+ * Custom Field class.
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class FieldController extends \Joomla\CMS\MVC\Controller\BaseController
+{
+
+	/**
+	 * Method to save a field
+	 *
+	 * @return  void
+	 *
+	 * @since __DEPLOY_VERSION__
+	 */
+	public function FEInlineEdition()
```

Comment:
Set return type:
`public function FEInlineEdititing(): void`

Remember we change `Edition` to `Editing`